Development.podspec

```diff
@@ -141,4 +142,25 @@ Pod::Spec.new do |s|
     test_spec.dependency 'WorkflowTesting'
     test_spec.dependency 'WorkflowRxSwiftTesting'
   end
+
+  # TODO: Disabled because app specs cannot increase the deployment target of the root
```

Comment:
I did go down that route at some point, which then `pod gen` was expecting EVERYTHING with the `_iOS13` suffix, which I wasn't sure how to handle without duplicating everything or rewriting `pod gen` to be able to handle the special cases

Reply:
Gotcha!

Reply:
Yeah I looked into this a tiny bit when I first started at Square. I can confirm the comment is correct, and don't remember how difficult it would be to fix